### ISSUE TITLE
[Feature Request]: Ability to zoom in on or enlarge displayed images

### ISSUE THREAD

**Issue by s-truax** [karolzak/ipyplot#9]
It would be great if, after displaying a grid of images, I could click on an individual image to zoom in on it.

**Comment by karolzak** [comment]
Hi @s-truax 
That's a really good idea! I'll try to include it in the next release.
Stay tuned!

**Comment by karolzak** [comment]
Hi @s-truax 
I already have an early version of this feature for testing.
You can install it by using this command:
```
pip install git+https://github.com/karolzak/ipyplot.git@image-enlarge --upgrade
```

Try it out when you have a chance and let me know if you have any feedback.
Thanks!

**Comment by karolzak** [comment]
Implemented, tested and pushed in with #13 
Feature will be released with 1.1 version #4
